CONTRIBUTING.md

```diff
@@ -7,36 +7,30 @@ please read the [code of conduct](CODE_OF_CONDUCT.md).
 
 > Install yarn on your system: [https://yarnpkg.com/en/docs/install](https://yarnpkg.com/en/docs/install)
 
+> Install lerna globally: `yarn global add lerna`
+
 ```sh
 $ git clone https://github.com/netlify/netlify-cms
 $ cd netlify-cms
-$ yarn
+$ lerna bootstrap
```

Comment:
@erquhart Looks like you also have a `bootstrap` script in the package.json, do we really need to run Yarn before Lerna, since Lerna is supposed to do it itself.

Reply:
Ah, interesting, I didn't notice that. Going that route removes the need to have lerna installed globally too. Or at least it should.

Reply:
Yeah, then you know it is using the relevant version of Lerna as well (the one our project is set up with).